backend/db/migrate/20250113000001_add_name_and_image_to_users.rb

```
@@ -0,0 +1,6 @@
+class AddNameAndImageToUsers < ActiveRecord::Migration[8.0]
```

RuboCop (1.76.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)